src/function/function.cpp

```diff
@@ -99,6 +99,7 @@ void BuiltinFunctions::Initialize() {
 	RegisterGenericFunctions();
 	RegisterOperators();
 	RegisterSequenceFunctions();
+	RegisterDebugFunctions();
```

Comment:
We have shifted away from this method of function registration for most functions - see https://github.com/duckdb/duckdb/pull/7310